janusgraph-grpc/src/main/proto/janusgraph/types/v1/schema_types.proto

```diff
@@ -21,6 +21,54 @@ import "google/protobuf/wrappers.proto";
 option java_multiple_files = true;
 option java_package = "org.janusgraph.graphdb.grpc.types";
 
+enum PropertyDataType {
+    // java class: String.class
+    STRING = 0;
```

Comment:
Usually by convention it is suggested to use `UNSPECIFIED` element for `0` enum value.
https://developers.google.com/protocol-buffers/docs/style#enums

Do you think we should follow this style convention? I don't have preference but thought could be good to discuss it

Reply:
fixed